testgrid/config.yaml

```diff
@@ -7450,3 +7461,4 @@ dashboard_groups:
   - vmware-presubmits-cloud-provider-vsphere
   - vmware-conformance
   - vmware-conformance-cloud-provider
+  
```

Comment:
trailing space? 😂

Reply:
^^ (lgtm otherwise)

Reply:
resolved, sorry it slipped my mind. lots to do